### ISSUE TITLE
PJM data download issue in Data Manager

### ISSUE THREAD

**Issue by atribera** [snl-quest/snl-quest#36]
I am unable to enter any input into the box designated for the Node ID. I am getting the following error when I try to download data without the Node ID but one of the four boxes checked (PJM average, Aggregate, Zone, Hub):

**At least one download thread failed. See the log for details. Please retry downloading data for the months that returned errors.**

**Comment by AlvaroFurlani** [comment]
This field accepts only numeric characters.

@atribera you may close this issue after you test again to make sure it works.

**Comment by atribera** [comment]
Yes, it works when I use numeric characters. However, there's another problem I am facing now. When I hit the download button after filling out all the necessary input fields, I see the following:

```
**ValueError('Mixing dicts with non-Series may lead to ambiguous ordering.')**
[INFO   ] [PJMdownloader] Number of nodes in this call: 1.
**[ERROR  ] [PJMdownloader] 201401, 1, lmp: An unexpected error has occurred. (UnboundLocalError("local variable 'total_nrows' referenced before assignment"))**
[INFO   ] [PJMdownloader] 201401, n/a, reg: Successfully downloaded.
[INFO   ] [PJMdownloader] 201401, n/a, mileage: Successfully downloaded.
[WARNING] [PJMdownloader] At least one download thread failed. See the log for details. Please retry downloading data for the months that returned errors.
```
I see in my data folder that the data required for frequency regulation have been successfully download but the LMPs have not. 

In addition, at PJM's data miner website it shows only two types of nodes: ZONE and LOAD. However, in the data manager we see four types: PJM (average), Aggregate, Zone and Hub. Can you please clarify this?
